app.py

```diff
@@ -0,0 +1,74 @@
+from flask import Flask, render_template, request, redirect, url_for, jsonify, flash
+import mysql.connector
+
+app = Flask(__name__)
+app.secret_key = '\xbd\xa4\xe9_4[!\xe5oW\xc9\xfc\xdfM+\xaf\x9b\x88my\xd6\xfe\xd9T'  # Replace with your own secret key
+
+# Database connection
+db = mysql.connector.connect(
+    host='localhost',
+    user='root',
+    password='Bpr123',
+    database='mysqll'
```

Comment:
Another issue is that you have to create table manually with the required fields. Ideally you application should have all the code defined for it. 

Try to explore SqlAlchemy for flask and create and manage tables. 

For example in the below code you can create your table a object model and use it in the code, instead of calling cursor all the time.

```
from flask_sqlalchemy import SQLAlchemy
app.config['SQLALCHEMY_DATABASE_URI'] = 'mysql://username:password@localhost/your_database_name'
app.config['SQLALCHEMY_TRACK_MODIFICATIONS'] = False
db = SQLAlchemy(app)
class Task(db.Model):
    __tablename__ = 'tasks'
    
    id = db.Column(db.Integer, primary_key=True, autoincrement=True)
    title = db.Column(db.String(255), nullable=False)
    description = db.Column(db.String(100), nullable=False)
    complete = db.Column(db.Boolean, default=False, nullable=False)

@app.before_first_request
def create_tables():
    db.create_all()
```